docs/doc/14-sql-commands/40-show/show-settings.md

```diff
@@ -22,6 +22,7 @@ SHOW SETTINGS;
 | name                                  | value        | default      | level   | description                                                                                                                                                                         | type   |
 +---------------------------------------+--------------+--------------+---------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
 | collation                             | binary       | binary       | SESSION | Sets the character collation. Available values include "binary" and "utf8".                                                                                                         | String |
+| disable_json_check                    | 0            | 0            | SESSION | Disable check JSON syntax, allow insert invalid JSON.                                                                                                                               | UInt64 |
```

Comment:
format options or `settings` ? cc @youngsofun

Reply:
for safety:    
   copy option (affect a copy OP) >  format options( all copies using this format) > settings (may affect other irrelevant copies in the same session)

Reply:
using format options?

Reply:
using `copy option` more reasonable to me

e.g.

CSV row `abc` can be loaded to  table1(string) or table2(variant)
i.e. disable_json_check is meaningful only for table2, 
in other words, `disable_json_check` is not a property of the CSV format

Reply:
using `copy option` more reasonable to me

e.g.

CSV row `abc` can be loaded to  table1(string) or table2(variant)
i.e. disable_json_check is meaningful only for table2, 
in other words, `disable_json_check` is not a property of the CSV format